Resources/Prototypes/_Carpmosia/Maps/saltern-2.yml

```
@@ -0,0 +1,65 @@
+- type: gameMap
+  id: Saltern2
+  mapName: 'Saltern-II'
+  mapPath: /Maps/_Carpmosia/saltern-2.yml
+  minPlayers: 0
+  maxPlayers: 35
```

God forbid.
```suggestion
  maxPlayers: 20
```
Can drop this to 15 even.